characters/update_feats_table.py

```diff
@@ -102,6 +137,74 @@ def add_feat(feat):
         cursor.close()
         connection.commit()
 
+""" prompt to change a feat in the database to match what's on the page """
+def update_a_feat(doc_feat, psql_feat):
+    print '\n\n\n'
```

Comment:
In Java there's a function called System.lineSeparator() that prints whatever the running OS's newline character is. Does python have an equivalent?